src/components/Pages/ManageWiki/Features/FederatedProperties.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <v-form ref="form" v-model="valid">
-    <v-card color="#fad1d0">
+    <v-card disabled tile elevation="2" class="padding" outlined>
       <v-card-title>Federated properties</v-card-title>
+      <v-card-subtitle><br/>(currently not available)</v-card-subtitle>
       <v-card-text>
       Federated properties allows you to use the properties of Wikidata on your Wikibase instance.<br>
-      <span v-if="isEnabled">If you turn this off, and confirm the warning, federated properties used on your wikibase will no longer be accessible!</span>
-      <span v-else>If you turn this on, and confirm the warning, any existing properties on your wikibase will no longer be accessible!</span>
       </v-card-text>
       <v-card-actions>
         <v-btn @click="showModal" color="red" v-if="isEnabled">Disable Federated Properties</v-btn>
-        <v-btn @click="showModal" color="red" v-else>Enable Federated Properties</v-btn>
+        <v-btn disabled elevation="2" @click="showModal" v-else>Enable Federated Properties</v-btn>
```

Comment:
My understanding of the task is that we want to disable the feature not remove code. I think we can remove `@click` on the disabled button but I am a bit unsure about deleting code causing enabling it later in the future to result to rewriting the same code.

Reply:
We had a talk about this and decided that rather than maintaining unreachable code we would be happier to just pull it back our of git in the event that we need it